packages/alchemy/src/react/hooks/useLogout.ts

```diff
@@ -0,0 +1,46 @@
+"use client";
+
+import { useMutation, type UseMutateFunction } from "@tanstack/react-query";
+import { useAlchemyAccountContext } from "../context.js";
+import type { BaseHookMutationArgs } from "../types.js";
+import { useSigner } from "./useSigner.js";
+
+export type UseLogoutData = void;
```

Comment:
do we need this why not inline the void?

Reply:
yeah i was back and forth on this, can remove